config/rooms.cfg

```diff
@@ -35,6 +35,8 @@
 # LibraryCooldownWorkMin            (uint)   How many turns a creature will stay idle after working minimum
 # LibraryCooldownWorkMax            (uint)   How many turns a creature will stay idle after working maximum
 # LibrarySkillPointsBook            (uint)   How many points have to be researched to create a book
+# PortalCooldownSpawnMin            (uint)   How many turns a portal will not spawn any creature after spawning minimum
```

Comment:
As a user point of view, "... after spawning minimum" is not very straight-forward. I would simply say:
"How many minimum turns a portal will wait before spawning a creature"
and respectively:
"How many maximum turns a portal will wait before spawning a creature"

All that, if I understood the code correctly. ;)